### PR TITLE
UI welcome sing-in sign-up

### PR DESCRIPTION
**Pull request by baciu-radu**

### CODE REVIEW

**Review by AlinaSastras — APPROVED** [review]

**Review by adrianlibenciuc — APPROVED** [review]

File: src/main/java/us/icarenow/portal/controller/PatientController.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
+package us.icarenow.portal.controller;
```
Comment (andreimladin) [inline src/main/java/us/icarenow/portal/controller/PatientController.java:1]: Move it to API

File: src/main/java/us/icarenow/portal/controller/PatientController.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import org.springframework.web.bind.annotation.*;
+import us.icarenow.portal.model.Patient;
+import us.icarenow.portal.service.PatientService;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+public class PatientController {
+    public PatientService patientService = new PatientService();
```
Comment (andreimladin) [inline src/main/java/us/icarenow/portal/controller/PatientController.java:14]: `@Autowired
private PatientService patientService`

**Review by andreimladin — CHANGES_REQUESTED** [review]

File: src/main/java/us/icarenow/portal/service/PatientService.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
+package us.icarenow.portal.service;
+
+import us.icarenow.portal.model.Patient;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class PatientService {
```
Comment (andreimladin) [inline src/main/java/us/icarenow/portal/service/PatientService.java:9]: @Service

File: src/main/java/us/icarenow/portal/service/PatientService.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
+package us.icarenow.portal.service;
+
+import us.icarenow.portal.model.Patient;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
```
Comment (andreimladin) [inline src/main/java/us/icarenow/portal/service/PatientService.java:8]: @Service

**Pull request closed without merging**